internal/core/app_status_report.go

```diff
@@ -241,11 +242,9 @@ func (op *statusReportOperation) Init(app *App) (proto.Message, error) {
 	return &pb.StatusReport{
 		Application: app.ref,
 		Workspace:   app.workspace,
-		ResourcesHealth: []*pb.StatusReport_Health{
-			{
-				HealthStatus:  "UNKNOWN",
-				HealthMessage: "Unknown health status",
-			},
+		Health: &pb.StatusReport_Health{
```

Comment:
Part of deprecating ResourcesHealth - this field wasn't used I don't think. I like the idea of starting the whole status report in state unknown though, so i'm moving that to the top level `Health`.